src/pages/manage/productions/create.js

```diff
@@ -1,28 +1,314 @@
+      maxWidth="43rem"
+      label={t('productions.create.production_name')}
+      emptyLabel={t('productions.create.no_productions')}
+      onInput={onInput}
+      onSelection={onSelection}
+      {...getBoxProps(props)}
+    />
+  );
+};
+
+ZeroOrOneProduction.propTypes = {
+  productionName: PropTypes.string,
+  suggestedProductions: PropTypes.array,
+  onInput: PropTypes.func,
+  onSelection: PropTypes.func,
+  className: PropTypes.string,
+};
+
+ZeroOrOneProduction.defaultProps = {
+  suggestedProductions: [],
+};
+
+const TwoProductions = ({ productionNames, onChange, ...props }) => {
+  const { t } = useTranslation();
+  return (
+    <RadioButtonGroup
+      name="production-names"
+      items={productionNames}
+      groupLabel={t('productions.create.production_name')}
+      onChange={onChange}
+      {...getStackProps(props)}
+    />
+  );
+};
+
+TwoProductions.propTypes = {
+  productionNames: PropTypes.array,
+  onChange: PropTypes.func,
+};
+
+TwoProductions.defaultProps = {
+  productionNames: [],
+};
+
+const LinkAndSkipButtons = ({
+  shouldDisableLinkButton,
+  onClickLink,
+  onClickSkip,
+  ...props
+}) => {
+  const { t } = useTranslation();
+  return (
+    <Inline spacing={3} {...getInlineProps(props)}>
+      <Button
+        variant={ButtonVariants.SUCCESS}
+        disabled={shouldDisableLinkButton}
+        onClick={onClickLink}
+      >
+        {t('productions.create.link')}
+      </Button>
+      <Button variant={ButtonVariants.DANGER} onClick={onClickSkip}>
+        {t('productions.create.skip')}
+      </Button>
+    </Inline>
+  );
+};
+
+LinkAndSkipButtons.propTypes = {
+  shouldDisableLinkButton: PropTypes.bool,
+  onClickLink: PropTypes.func,
+  onClickSkip: PropTypes.func,
+};
+
+const ProductionStatus = {
+  MISSING: 'missing',
+  EXISTING_SELECTED: 'existingSelected',
+  NEW: 'new',
+};
 
 const Create = () => {
-  return <Title>Productions - create</Title>;
-};
-
-export const getServerSideProps = async ({ req, query }) => {
-  const { cookies } = new Cookies(req?.headers?.cookie);
-  const isUnAuthorized = !cookies.token && !query?.jwt;
-
-  if (isUnAuthorized) {
-    return {
-      redirect: {
-        destination: '/login',
-        permanent: false,
-      },
-    };
-  }
-
-  return {
-    props: {
-      cookies,
-    },
+  const { t, i18n } = useTranslation();
+  const queryClient = useQueryClient();
+  const [searchInput, setSearchInput] = useState('');
+  const [selectedProductionId, setSelectedProductionId] = useState('');
+
+  const {
+    data: suggestedEvents,
+    status: suggestedEventsStatus,
+    refetch: refetchSuggestedEvents,
+  } = useGetSuggestedEvents();
+  const { data: suggestedProductionsData } = useGetProductions({
+    name: searchInput,
+    limit: 10,
+  });
+
+  const handleSuccess = async () => {
+    setSelectedProductionId('');
+    setSearchInput('');
+    await refetchSuggestedEvents();
+    await queryClient.refetchQueries(['productions']);
```

Comment:
I would not use refetchQueries here, this will retrigger all caches that have 'productions' in it. I would use the refetch function that is returned from useGetProductions()